resources/views/ceklulus.blade.php

```diff
@@ -1,160 +1,158 @@
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet">
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/js/bootstrap.bundle.min.js"></script>
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@600&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Electrolize&family=Fira+Code:wght@400&display=swap" rel="stylesheet">
+
 </head>
-<body>
 
-@include('layouts/navbarCaasLogout')
-@section('navcaaslogout')
-@endsection
+<body style="background-color: #2c4d16;">
 
-<section id="main-nim">
-  @if($Active==1 && $urut_tahap==$current_tahap)
-    <div class="container p-5 pt-0">
-      <div class="d-flex justify-content-center pb-4">
-        <div>
-          <img src="{{asset('/assets/dlor-blue.png')}}" alt="logo" class="dlor-logo-announce">
-        </div>
-      </div>
+  @include('layouts/navbarCaasLogout')
+  @section('navcaaslogout')
+  @endsection
+
+  <section id="landing-section">
+    @if($Active===1 && $urut_tahap===$current_tahap)
+    <div class="container">
       <div class="d-flex justify-content-center">
-        <div style="background-color: #FFC844" class="checker-box pb-5">
+        <div class="checker-box">
           <div class="text-center text-nim-head">
-            <span>Apakah kamu lulus Tahap {{ $namatahap }}?</span>  
+            <span>Apakah kamu lulus Tahap {{ $namatahap }}?</span>
           </div>
           <div class="text-center pt-3 pb-3">
             <div>
-              <span style="font-weight: 600" class="uppercase Welcome-text">
-              {{$nama}}
+              <span class="uppercase text-nim">
+                {{$nama}}
+              </span>
+            </div>
+            <div>
+              <span class="text-nim">
+                NIM : {{$nim}}
+              </span>
+            </div>
+            @if($isLolos==1 && $urut_tahap==$current_tahap)
+            <div class="pt-3">
+              <span style="font-size:25px;font-weight: 700;color:green;" class="Welcome-text">
+                {!!nl2br(e($lulustext))!!}
+              </span>
+            </div>
+            <div>
+              @if($linktext!='-')
+              <div class="d-flex justify-content-center pt-3">
+                <a style="text-decoration: none" href="{{$linktext}}" target="_blank">
+                  <button style="border-radius: 20px;" class="button-submit">
+                    <div>
+                      <div>
+                        CLICK THIS LINK
+                      </div>
+                    </div>
+                  </button>
+                </a>
+              </div>
+              @endif
+            </div>
+            @else
+            <div class="pt-3">
+              <span style="font-weight: 700;color:red;" class="Welcome-text">
+                {!!nl2br(e($failedtext))!!}
```

Comment:
kalau ini sih masih aman kayaknya, tapi kl emang samar nanti diubah aja

Reply:
ini untuk tidak lolos bang :
![image](https://user-images.githubusercontent.com/88444330/160158090-2bb4be9f-a559-44ab-aca3-053418a29e47.png)

kalau misal saya ubah jadi gini gimana bang? biar disamain seperti kondisi lolos
```suggestion
            <span style="font-size:2.5vw;font-weight: 700;color:whitesmoke;background-color:#FF1212;border-radius:17px; padding:10px 15px 10px 15px" class="Welcome-text">
                {!!nl2br(e($failedtext))!!}
```
![image](https://user-images.githubusercontent.com/88444330/160158320-e92ec1ad-f432-4cef-9113-fc6a0a4bdd72.png)

Reply:
nah iyaa mantapp

Reply:
oke saya push ya bang

Reply:
oke siapp